src/System.Runtime.Loader/tests/DefaultContext/System.Runtime.Loader.DefaultContext.Tests.csproj

```diff
@@ -19,5 +19,8 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </ProjectReference>
   </ItemGroup>
+  <Target Name="RenameTestAssembly" AfterTargets="AfterBuild">
```

Comment:
@AlexGhiondea the idea is that AssemblyName must not match the dll name and also other projects here use this reference and they do not expect renamed assembly